### DISCUSSION TITLE
Can I call ExtPay outside of the extension?

### DISCUSSION THREAD

**Discussion by bergwerf** [Glench/ExtPay#151] [General]
When a user uninstalls my extension, I want to remind them to cancel their subscription on the webpage that I set via `runtime.setUninstallURL`. Could I redirect them to their subscription status page from my website, such that they would not have to install the extension again to cancel?

**Comment by Glench** [comment]
Sure! You can take a look at the ExtPay source code to see the URL that's generated, although at the moment access to this URL is not guaranteed to be the same across versions so beware that it might change in the future.
